### PR TITLE
Align widget display modes

### PR DESCRIPTION
**Pull request by mathturtle**
Taking a stab at #60.  @cryos How is this?

### CODE REVIEW

**Comment by mathturtle** [comment]
For waffle.io: resolves #60.

**Comment by Hovden** [comment]
Can we merge?

File: tomviz/AlignWidget.h [hunk 1]
```diff
@@ -77,34 +78,10 @@ protected slots:
 
   void sliceOffsetEdited(int slice, int offsetComponent);
 
-protected:
```
Comment (cryos) [inline tomviz/AlignWidget.h]: I would rather we avoided moving everything to an internals class, we are not worried about inheritance/minimizing header includes. I would like to stick with normal members unless there is a good reason to switch.

**Comment by cryos** [comment]
We can merge, I would like to stop moving everything to an internals class unless we have a compelling reason, but other than that this patch looks fine at first glance (I haven't really gone through the view modes, or tested out the UI improvements, but we can iterate on it once merged too).

**Comment by mathturtle** [comment]
Updated.  I tested the new version and it still works for me.

**Comment by cryos** [comment]
Thanks Shawn

**Pull request merged**